mlrun/utils/helpers.py

```diff
@@ -983,6 +983,12 @@ def get_workflow_url(project, id=None):
     return url
 
 
+def is_igz_version_sufficient(required_version):
```

Comment:
no need for a new method you can use `mlrun.platforms.iguazio.min_iguazio_versions` or `mlrun.utils.helpers.validate_component_version_compatibility`

Reply:
nice thx 👍